persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/table/management/AbstractTableManager.java

```diff
@@ -173,17 +184,20 @@ public void executeUpdateSql(Connection conn, String sql) throws PersistenceExce
 
    public void dropTable(Connection conn) throws PersistenceException {
       dropTimestampIndex(conn);
-      String dropTableDdl = "DROP TABLE " + getTableName();
+
       String clearTable = "DELETE FROM " + getTableName();
       executeUpdateSql(conn, clearTable);
+
+      String dropTableDdl = "DROP TABLE " + getTableName();
       if (log.isTraceEnabled()) {
          log.tracef("Dropping table with following DDL '%s'", dropTableDdl);
       }
       executeUpdateSql(conn, dropTableDdl);
    }
 
+   // TODO: need to change to drop any index
```

Comment:
I think we need this as part of this PR

Reply:
Yeah I need to do that, thanks for reminding me.